<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartSetting1</class>
 <widget class="QDialog" name="ChartSetting1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图表设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/* 仅保留对话框基础字体和颜色设置，具体的CheckBox样式已移至main.cpp全局管理 */
QDialog {
    background-color: #ffffff;
    color: #000000;
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 10pt;
}

/* 按钮样式优化 */
QPushButton {
    border: 1px solid #8f8f91;
    border-radius: 4px;
    background-color: #f5f5f5;
    color: #000000;
    padding: 5px 15px;
}
QPushButton:hover {
    background-color: #e0e0e0;
}
QPushButton:pressed {
    background-color: #d0d0d0;
}
QPushButton:default {
    border: 1px solid #0078d7;
    color: #0078d7;
    font-weight: bold;
}

/* 表格样式 */
QTableWidget {
    selection-background-color: #0078d7;
    selection-color: #ffffff;
    gridline-color: #d0d0d0;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>标题设置</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="labelTitle">
         <property name="text">
          <string>图表标题:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editTitle"/>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkTitleVisible">
         <property name="text">
          <string>显示标题</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabX">
      <attribute name="title">
       <string>X轴设置</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>轴标签:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editXLabel"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>范围最小值:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="spinXMin">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-999999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>范围最大值:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="spinXMax">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-999999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkXLog">
         <property name="text">
          <string>对数坐标 (Logarithmic)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="checkXSci">
         <property name="text">
          <string>科学计数法显示</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="checkXGrid">
         <property name="text">
          <string>显示主网格</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="checkXSubGrid">
         <property name="text">
          <string>显示子网格</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabY">
      <attribute name="title">
       <string>Y轴设置</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>轴标签:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="editYLabel"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>范围最小值:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="spinYMin">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-999999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>范围最大值:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="spinYMax">
         <property name="decimals">
          <number>6</number>
         </property>
         <property name="minimum">
          <double>-999999999.000000000000000</double>
         </property>
         <property name="maximum">
          <double>999999999.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="checkYLog">
         <property name="text">
          <string>对数坐标 (Logarithmic)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="checkYSci">
         <property name="text">
          <string>科学计数法显示</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="checkYGrid">
         <property name="text">
          <string>显示主网格</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="checkYSubGrid">
         <property name="text">
          <string>显示子网格</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCurves">
      <attribute name="title">
       <string>图例设置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_Curves">
       <item>
        <widget class="QLabel" name="label_Curves">
         <property name="text">
          <string>曲线列表 (双击名称以修改):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableGraphs">
         <property name="columnCount">
          <number>3</number>
         </property>
         <column>
          <property name="text">
           <string>样式</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>曲线名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>显示</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
